website/documentation/getting-started/architecture/_index.md

```diff
@@ -0,0 +1,75 @@
+
+You can connect to the Gardener API Endpoint (i.e., the API server in the garden cluster) either through the dashboard or with kubectl, given that you have a proper kubeconfig for it.
+
+### The Seeds Running the Shoot Cluster Control Planes
+
+Inside each seed is one of the most important controllers in Gardener - the gardenlet. It spawns many other controllers, which will eventually create all resources for a shoot cluster, including all resources on the cloud providers such as virtual networks, security groups, and virtual machines.
+
+## Gardener's API Endpoint
+
+Kubernetes' API can be extended - either by CRDs or by API aggregation.
+
+![](./images/api-endpoint-1.png)
+
+API aggregation involves setting up a so called extension-API-server and registering it with the main Kubernetes API server. The extension API server will then serve resources of custom-defined API groups on its own. While the main Kubernetes API server is still used to handle RBAC, authorization, namespacing, quotas, limits, etc.,  all custom resources will be delegated to the extenstion-API-server. This is done through an APIService resource in the main API server - it specifies that, e.g., the API group `core.gardener.cloud` is served by a dedicated extension-API-server and all requests concerning this API group should be forwarded the specified IP address or Kubernetes service name. Extension API servers can persist their resources in their very own etcd but they do not have to - instead, they can use the main API servers etcd as well.
+
+![](./images/api-endpoint-2.png)
+
+Gardener uses its very own extension API server for its resources like Shoot, Seed, CloudProfile, SecretBinding, etc... However, Gardener does not set up a dedicated etcd for its own extension API server - instead, it reuses the existing etcd of the main Kubernetes API server. This is absolutely possible since the resources of Gardener's API are part of the API group `gardener.cloud` and thus will not interfere with any resources of the main Kubernetes API in etcd.
+
+In case you are interested, you can read more on:
+- [API Extension](https://kubernetes.io/docs/tasks/extend-kubernetes/setup-extension-api-server/)
+- [API Aggregation ](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation)
+- [APIService Resource](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/#register-apiservice-objects)
+
+## Gardener API Resources
+
+![](./images/api-resources.png)
+
+Gardener's API endpoint serves many different resources.
+
+Since Gardener's API endpoint is a regular Kubernetes cluster, it would theoretically serve all resources from the Kubernetes core API, including Pods, Deployments, etc. However, Gardener implements RBAC rules and disables certain controllers that make these resources virtually inaccessible. Objects like Secrets, Namespaces, and ResourceQuotas are still available, though, as they play a vital role in Gardener.
```

Comment:
```suggestion
Since Gardener's API endpoint is a regular Kubernetes cluster, it would theoretically serve all resources from the Kubernetes core API, including Pods, Deployments, etc. However, Gardener implements RBAC rules and disables certain controllers that make these resources inaccessible. Objects like Secrets, Namespaces, and ResourceQuotas are still available, though, as they play a vital role in Gardener.
```